### PR TITLE
needed to update megaparsec for corba

### PR DESCRIPTION
**Pull request by don41382**
!@charleso @nhibberd @jystic 

update for corba was required

see
> https://github.com/ambiata/technology/issues/552
/jury approved @jystic

### CODE REVIEW

File: icicle-compiler/icicle.cabal [hunk 1]
```diff
@@ -88,7 +88,7 @@ library
                      -- a bit cleaner.
                      -- As an added benefit, parsec will give us nicer error messages.
                      , attoparsec                      >= 0.12       && < 0.14
-                     , megaparsec                      == 5.0.*
+                     , megaparsec                      == 5.1.*
```
Comment (jystic) [inline icicle-compiler/icicle.cabal]: Can we just increase the upper bound here? bumping the lower bound could break things that depend on icicle.

```megaparsec                      >= 5.0        && < 5.2```

Comment (don41382) [reply icicle-compiler/icicle.cabal]: I'll do that

**Comment by don41382** [comment]
@jystic are you fine with this?

**Comment by jystic** [comment]
👍 yep, all good now

**Comment by etorreborre** [comment]
Can I ask what is `corba`? This doesn't seem to be a public Ambiata project and the name brings some painful memories :-).

**Pull request merged**